client/control/components/RecipeList.js

```diff
@@ -169,6 +233,14 @@ class DisconnectedRecipeList extends React.Component {
   }
 }
 
+DisconnectedRecipeList.propTypes = {
```

Comment:
Fair. I think `static propTypes = { ... }` would have also satisfied this. I'm not too picky either way. Do what you prefer.